docs/data/toolpad/connecting-to-data-sources/connections.md

```diff
@@ -0,0 +1,86 @@
+# Connections
+
+<p class="description">
+    <b>CONNECTION</b> instance allows us to reuse certain configuration details to create <b>queries</b> faster
+</p>
+
+In order to create a new `CONNECTION` instance press `+` button in the `Instance editor` on the left
+
+![Add Connection](/static/toolpad/connection.png)
+
+## Connection types
```

Comment:
In terms of organizing the information, I wonder if it makes sense to group more by datasource. e.g. instead of:

```
* queries:
  * shared info
  * datasources
    * function
    * postgres
    * fetch
    * sheets
* connections:
  * shared info
  * datasources
    * function
    * postgres
    * fetch
    * sheets
```

More towards

```
* queries (shared info)
* connections (shared info)
* datasources
  * function
    * connection options
    * query options
  * postgres
    * connection options
    * query options
  * fetch
    * connection options
    * query options
  * sheets
    * connection options
    * query options
```

It's just a bit strange that if I want to set up a postgres query, I'd have to visit two different pages. First I have to find the postgres section in `connections.md`. And then I have to visit the postgres.md page for the actual query.

Reply:
The reason why I split them up initially is that it's how it's currently done in the app itself. We do have 2 separate concepts - connection and datasource. I don't really see what we could put as a shared content for `Connection` as each connection type is configured differently. In addition - once you setup connection you should no longer need access to it's docs.
Anyways, I'm not suggesting to close this conversation, but maybe we could aim to merge this PR and then once we have a final call on the structure for this, do necessary adjustments?